cic_dev/addon/pre-requisites/launch-early

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+a=`grep -ir "Alias=rc-local.service" /etc/systemd/system/rc-local.service`
+b=`grep -ir "# start aic" /etc/rc.local`
+
+if [ -z "$a" -a -z "$b" ] #if script run before -- START
+then
+echo "Adding launch-early"
+sudo ln -fs /lib/systemd/system/rc-local.service /etc/systemd/system/rc-local.service
+cat << EOF | sudo tee -a /etc/systemd/system/rc-local.service 
+[Install]  
+WantedBy=multi-user.target  
+Alias=rc-local.service  
+EOF
+
+sudo touch /etc/rc.local
+cat << 'EOF' | sudo tee /etc/rc.local 
+#!/bin/sh
+# start aic
+while (( $(ps -ef | grep -v grep | grep dockerd | wc -l) == 0 )); do sleep 1; done
+echo "PASSWD" | sudo -Sk /AIC_WORK_DIR/aic start
+exit 0
+EOF
+sudo chmod +x /etc/rc.local
+
+sudo sed -i "s|PASSWD|$password|g" /etc/rc.local
```

Comment:
Save password in rc.local should have security issues. We'll merge a change soon which will allow start aic without root permission. Please remember to remove this later.

Reply:
save password is not good. must it be start before user login?

Reply:
I have removed this step now.
Without sudo rc.local is executed fine. I verified on my local EB.
Please have a check.